lib/node/pl-middle-layer/src/js_render/index.ts

```diff
@@ -39,7 +39,8 @@ export function computableFromRF(
     const vm = scope.manage(runtime.newContext());
     const rCtx = new JsExecutionContext(scope, vm,
       (s) => { deadlineSettings = s; },
-      codeWithInfo.sdkVersion,
+      sdkVersion,
+      featureFlags,
```

Comment:
sdkVersion it's prev solution ? Should we remove it?

Reply:
Yap, can be removed.